javascript-methods/main.js

```diff
@@ -0,0 +1,80 @@
+const x = 10;
+const y = 8;
+const z = 7;
+
+const maximumValue = Math.max(x, y, z);
+console.log(`maximumvalue: ${maximumValue}`);
+const heroes = ['jack', 'brett', 'IDK'];
+
+// assign a random number
+let randomNumber = Math.random();
+// append the length of the heroes array to randomNumber
+randomNumber *= heroes.length;
+console.log(randomNumber);
+// gets the index placements of the heroes array randomized
+const randomIndex = Math.floor(randomNumber);
+console.log(randomIndex);
+
+console.log(`randomIndex: ${randomIndex}`);
+
+const randomHero = heroes[randomIndex];
+
+console.log(`randomHero: ${randomHero} is a hero`);
+
+// Array Methods
+// array
+// book objects
+
+const js = {
+  title: 'JavaScript for Impatient Programmers',
+  author: 'Dr. Axel Rauschmayer',
+};
+const css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou',
+};
+
+const library = [
+  {
+    title: 'JS for intermidiate jobs',
+    author: 'Jack Almadjian',
+  },
+  {
+    title: 'CSS animations',
+    author: 'Alex Montoya',
+  },
+  {
+    title: 'Keyframes and 3d css styling',
+    author: 'shawn idk',
+  },
+];
+
+const lastBook = library.pop();
+
+console.log('lastBook:', lastBook);
+
+console.log('firstBook:', library[0]);
```

Comment:
This still needs to be addressed.

Instead of logging `library[0]`, you should be using the `shift` method on `library`, and save that to a variable and log that value instead. We want to showcase how you can use methods to get these indexes as well